Source/dom/models/Asset.js

```diff
@@ -0,0 +1,126 @@
+import { isNativeObject } from 'util'
```

Comment:
maybe instead of having one big file with 3 definitions, we can create a folder `Asset` with a file `index.js` exporting the `ColorAsset` and `GradientAsset` defined in their own file

Reply:
Are you thinking that asset level would be in `/models` or on the same level as `/style`?

Reply:
I'd put it on the same level as `/style` but I really don't mind either

Reply:
Went ahead and moved everything to `/assets` similar to `/layers`.